app/i18n/it/sub.php

```diff
@@ -138,12 +138,12 @@
 				'xpath' => 'XPath per:',
 			),
 			'json_dotnotation' => array(
-				'_' => 'JSON (dot notation)',	// TODO
+				'_' => 'JSON (notazione a punti)',
 				'feed_title' => array(
 					'_' => 'titolo feed',
 					'help' => 'Esempio: <code>meta.title</code> o una stringa statica: <code>"Il mio feed personalizzato"</code>',
 				),
-				'help' => 'A JSON dot notated uses dots between objects and brackets for arrays (e.g. <code>data.items[0].title</code>)',	// TODO
+				'help' => 'Una notazione JSON utilizza i punti tra gli oggetti e le parentesi per gli array (p.es. <code>data.items[0].title</code>)',
```

Comment:
```suggestion
				'help' => 'Una notazione JSON utilizza i punti tra gli oggetti e le parentesi per gli array. P.es. <code>data.items[0].title</code>',
```

Could that be acceptable? For technical reasons, `array (` is not supported at the moment inside a sentence

Reply:
@Alkarex I just translated I didn't want to change the formatting of the text so I found it so I translated it